docs/next/modules/en/nav.adoc

```diff
@@ -1,55 +1,27 @@
-*** xref:getting-started/cluster-class/intro.adoc[Introduction]
-*** xref:getting-started/cluster-class/create_cluster.adoc[Create a cluster using Fleet]
-** xref:getting-started/air-gapped-environment.adoc[Air-gapped environment]
-** xref:getting-started/uninstall_turtles.adoc[Uninstall {product_name}]
-* Reference Guides
-** Architecture
-*** xref:reference-guides/architecture/intro.adoc[Introduction]
-*** xref:reference-guides/architecture/components.adoc[Components]
-*** xref:reference-guides/architecture/deployment.adoc[Deployment Scenarios]
-** xref:reference-guides/rancher-turtles-chart/values.adoc[Chart configuration]
-** CAPI Providers
-*** xref:reference-guides/providers/certified.adoc[Certified CAPI Providers]
-*** xref:reference-guides/providers/howto.adoc[Create & import a cluster using CAPI providers]
-*** xref:reference-guides/providers/addon-provider-fleet.adoc[Cluster API Addon Provider Fleet]
-** Test suite
-*** xref:reference-guides/test-suite/intro.adoc[Introduction]
-*** xref:reference-guides/test-suite/usage.adoc[Test suite guide]
-* Tasks
-** xref:tasks/intro.adoc[Introduction]
-** Cluster API Operator
-*** xref:tasks/capi-operator/basic_cluster_api_provider_installation.adoc[Basic Cluster API Provider Installation]
-*** xref:tasks/capi-operator/installing_core_provider.adoc[Installing the CoreProvider using CAPIProvider resource]
-*** xref:tasks/capi-operator/capiprovider_resource.adoc[CAPIProvider Resource]
-*** xref:tasks/capi-operator/clusterctlconfig_resource.adoc[ClusterctlConfig Resource]
-*** xref:tasks/capi-operator/add_infrastructure_provider.adoc[Installing AWS Infrastructure Provider using CAPIProvider resource]
-** Maintenance
-*** xref:tasks/maintenance/early_adopter_upgrade.adoc[Upgrade Instructions for Early Adopters]
-*** xref:tasks/maintenance/import_controller_upgrade.adoc[Upgrade Turtles import controller]
-*** xref:tasks/maintenance/automigrate_to_v3_import.adoc[Auto-migration to v3 cluster import]
-** Provider Certification
-*** xref:tasks/provider-certification/intro.adoc[What is a Certified Provider?]
-*** xref:tasks/provider-certification/process.adoc[Provider Certification Guide]
-* Developer Guide
-** xref:developer-guide/intro.adoc[Introduction]
-** xref:developer-guide/install_capi_operator.adoc[Installing Cluster API Operator]
-** xref:developer-guide/development.adoc[Development setup]
-** xref:developer-guide/contributing_guidelines.adoc[Guidelines]
+* xref:index.adoc[Overview]
+* Tutorials
+** xref:tutorials/quickstart.adoc[Quick Start]
+** xref:tutorials/first-cluster.adoc[Your First Cluster]
+** xref:tutorials/uninstall.adoc[Uninstall]
 * Reference
-** xref:reference/intro.adoc[Introduction]
+** xref:reference/architecture.adoc[Architecture]
 ** xref:reference/glossary.adoc[Glossary]
-** xref:reference/features.adoc[Features]
+** xref:reference/features.adoc[Feature]
+** xref:reference/certified.adoc[Certified Providers]
+* User guide
+** xref:user/capiprovider.adoc[CAPIProvider]
```

Comment:
Perhaps we should change the title for this page to something like `Install CAPI Providers` to make the goal/intent more clear. Unless the user already knows at this point what a `CAPIProvider` resource does.